src/replication/state.cpp

```diff
@@ -11,134 +11,258 @@
 
 #include "replication/state.hpp"
 
-#include "replication/status.hpp"  //TODO: don't use status for durability
+#include "replication/replication_server.hpp"
+#include "replication/status.hpp"
 #include "utils/file.hpp"
+#include "utils/variant_helpers.hpp"
 
 constexpr auto kReplicationDirectory = std::string_view{"replication"};
 
 namespace memgraph::replication {
 
+auto BuildReplicaKey(std::string_view name) -> std::string {
+  auto key = std::string{durability::kReplicationReplicaPrefix};
+  key.append(name.begin(), name.end());
```

Comment:
I think there is an overload for `string_view const &`, no need to specify begin() and end(), if you pass the entire view anyways,

Reply:
My IDE had issue with that, but the compiler is fine. I've made the change.